src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -403,7 +403,7 @@ def wait_for_create_job_to_complete(
     )
 
 
-def _hash_attachments(
+def hash_attachments(
```

Comment:
This method will be shared with other CLIs. I'm thinking if we should move it to `_common`.

I think I will just move it.